src/comm/DeviceProperties.h

```diff
@@ -16,8 +16,8 @@ static int64_t syclMaxWorkGroupSize(
   auto dev = q.get_device();
 
   auto kid = ::sycl::get_kernel_id<KernelClass>();
-  auto kbundle =
-      ::sycl::get_kernel_bundle<::sycl::bundle_state::executable>(ctx, {kid});
+  auto kbundle = ::sycl::get_kernel_bundle<::sycl::bundle_state::executable>(
+      ctx, {dev}, {kid});
```

Comment:
@fengyuan14 , could you help submit a GitHub issue to github.com/intel/llvm and then add some comments here to link the issue to be submitted?

Overall, LGTM.